src/app/pages/ElectronicDelivery.jsx

```diff
@@ -132,7 +138,7 @@ class ElectronicDelivery extends React.Component {
       itemId,
       itemSource,
       title,
-    } = this.state;
+    } = this.bibAndItemInformation();
```

Comment:
is the idea here that we aren't expecting these to change and trigger a rerender so we don't need them on the state?